rllib/offline/dataset_reader.py

```diff
@@ -142,7 +142,16 @@ def get_dataset_and_shards(
     # check paths to be a str or list[str] if not None
     if paths is not None:
         if isinstance(paths, str):
-            paths = [paths]
+            inputs = Path(paths).absolute()
+            if inputs.is_dir():
+                if format == "json":
+                    paths = list(inputs.glob("*.json")) + list(inputs.glob("*.zip"))
+                else:
+                    # we don't support zip files of parquet
+                    paths = list(inputs.glob("*.parquet"))
+                paths = [str(path) for path in paths]
+            else:
+                paths = [paths]
```

Comment:
why do we need to do this? doesn't datasets automatically glob the paths?

Reply:
does it! Its not mentioned in the documentation so I wasn't aware of that. I'll try that out sometime tomorrow

Reply:
ok I can confirm this is the case... that being said should we still merge the test or is it not worth it?

Reply:
gonna close this for now